fast_barcode_scanner/android/src/main/kotlin/com/jhoogstraat/fast_barcode_scanner/types/PreviewConfiguration.kt

```diff
@@ -1,5 +1,5 @@
 package com.jhoogstraat.fast_barcode_scanner.types
 
-data class PreviewConfiguration(val textureId: Long, val targetRotation: Int, val width: Int, val height: Int, val analysis: String) {
-    fun toMap() = hashMapOf("textureId" to textureId, "targetRotation" to targetRotation, "width" to width, "height" to height, "analysis" to analysis)
+data class PreviewConfiguration(val textureId: Long, val targetRotation: Int, val width: Int, val height: Int, val analysis: String, val analysisWidth: Int, val analysisHeight: Int) {
```

Comment:
This file could use some newlines.
Might be a good idea to run the formatter in Android Studio on all files.

Reply:
👍

Reply:
unfortunately, AS formatter doesn't have any opinions on formatting that file ☹️ ,  but I will fix it up manually.